extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/runtime/src/main/java/io/quarkus/resteasy/reactive/jackson/CustomSerialization.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.resteasy.reactive.jackson;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.lang.reflect.Type;
+import java.util.function.BiFunction;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+
+/**
+ * Annotation that can be used on RESTEasy Reactive Resource method to allow users to configure Jackson serialization
+ * for that method only, without affecting the global Jackson configuration.
+ */
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
Oh come on, you know we want to make this work on the class too ;) And package, and module!

Though we should only support the package case if we do for our other annotations too, but I doubt we do at this point.

The other annotations are generally method/class so let's be regular.

Reply:
To be honest, I'd rather keep it as limited as possible for the time being.

Reply:
If it's experimental, then mark it so ;)

Reply:
Good point!

https://media.giphy.com/media/rHR8qP1mC5V3G/giphy.gif